README.md

```diff
@@ -35,13 +35,13 @@ Goal: SDK size <= 1 MB and combined resources (object detection + regression mod
+Drishti is a [CMake](https://github.com/kitware/CMake) based project that uses the [Hunter](https://github.com/ruslo/hunter) package manager to download and build project dependencies from source as needed.  Hunter contains [detailed documentation](https://docs.hunter.sh/en/latest), but a few high level notes are provided here to help orient first time users.  In practice, some working knowledge of CMake may also be required.  Hunter itself is written in CMake, and is installed as part of the build process from a single `HunterGate()` macro at the top of the root `CMakeLists.txt` file (typically `cmake/Hunter/HunterGate.cmake`) (you don't have to build or install it).  Each CMake dependency's `find_package(FOO)` call that is paired with a `hunter_add_package(FOO CONFIG REQUIRED)` will be managed by Hunter.  Additional configuration (i.e., custom version and/or CMake arguments) can be performed for each package with an optional `hunter_config(FOO VERSION 1.0.0 CMAKE_ARGS ${FOO_CMAKE_ARGS}))` call in an optional local config file that is specified in the `HunterGate()` call.  In most cases, the only system requirement for building a Hunter project is a recent CMake with CURL support and a working compiler correpsonding to the operative toolchain.  Hunter will maintain all dependencies in a [versioned](https://docs.hunter.sh/en/latest/overview/customization.html) local [cache](https://docs.hunter.sh/en/latest/overview/shareable.html) by default (typically `${HOME}/.hunter`) where they can be reused in subsequent builds and shared between different projects.  They can also be stored in a server side [binary cache](https://docs.hunter.sh/en/latest/overview/binaries.html).
 
 The [Travis](https://github.com/elucideye/drishti/blob/master/.travis.yml) (Linux/OSX/iOS/Android) and [Appveyor](https://github.com/elucideye/drishti/blob/master/appveyor.yml) (Windows) CI scripts in the project's root directory can serve as a reference for basic setup when building from source.  To support cross platform builds and testing, the CI scripts make use of [Polly](https://github.com/ruslo/polly): a set of common CMake toolchains paired with a simple `polly.py` CMake build script.  Polly is used here for convenience to generate `CMake` command line invocations -- it is not required for building Hunter projects.
 
 To reproduce the CI builds on a local host, you must install `CMake` and `Polly` and add the paths for the `cmake` and `polly.py` executables to your system's `PATH` variable.  The `bin/hunter_env.{sh,cmd}` scripts (used in the CI builds) can be used to install these tools for you in your local terminal.  You may want to add the `PATH` variables permanently to your `.bashrc` file (or equivalent) for future sessions.
 
-Linux/OS X:
+Linux/OSX:
 ```
 source bin/hunter_env.sh
```

Comment:
Now it can be:
* Install CMake
* Install Python (for Polly)
* Clone Polly and add `<polly>/bin` to PATH

?

By the way "Install CMake" includes not only CMake but the real build tools and compiler. So probably this page can be helpful: http://cgold.readthedocs.io/en/latest/first-step.html

Reply:
I like the bullet list as a high level description.

> So probably this page can be helpful: http://cgold.readthedocs.io/en/latest/first-step.html

Perfect.  I'll add a link appropriately in the text.